frontend/src/app/features/login/login.component.scss

```diff
@@ -31,6 +50,20 @@
       justify-content: center;
       align-items: center;
       flex-direction: column;
+      button {
+        border-radius: 10px;
+        padding-left: 25px;
+        padding-right: 25px;
+        margin-bottom: 20px;
+        border: 2px solid white;
+        background-color: white;
+      }
+      button:disabled {
+        color: #c4c4c4;
+      }
+      button:enabled{
+        color: #364354;
+      }
```

Comment:
Nem lenne jobb globálisan beállítani vagy kivétel van?

Reply:
Kivétel volt ittis. 
![image](https://github.com/Hollo-Event-Notifier/Hollo-Event-Notifier-App/assets/43908750/403916e9-4a12-456e-b87d-ba962396660d)